primerpedia.js

```diff
@@ -38,10 +39,13 @@ function random() {
 }
 
 function search() {
+	setSearchButtonEnabledState();
```

Comment:
I'd prefer calling this `updateSeachButtonState()` (or the more verbose `updateSeachButtonEnabledState()`), and reserve "setX" names for functions that actually receive the parameter to set (or that indicate in their name what the target will be set to).

Reply:
Makes sense